src/data/evm_assets.json

```diff
@@ -3785,7 +4016,7 @@
           "transfer_fee": 0.1
         },
         {
-          "chain": "ethereum-sepolia",
```

Comment:
I think this is the wrong chain name and will break things. Can we figure out where the disconnect is? We've fixed this a couple times.

Reply:
the lowercased display name is being used here during generation. fixed that in the chains repo and #706